packages/service-clients/odsp-client/src/interfaces.ts

```diff
@@ -79,6 +82,22 @@ export interface OdspContainerAttachProps {
 	fileName: string | undefined;
 }
 
+/**
+ * Events emitted by the ODSP container service to notify consumers of select
+ * container changes.
+ * @beta
+ */
+export interface IOdspContainerServicesEvents extends IEvent {
```

Comment:
This interface should probably be documented as `@sealed`, since I'm assuming we don't intend for users to extend it.